ql/src/java/org/apache/hadoop/hive/ql/stats/BasicStatsTask.java

```diff
@@ -102,7 +102,6 @@ public BasicStatsTask(HiveConf conf, BasicStatsWork work) {
 
   @Override
   public int process(Hive db, Table tbl) throws Exception {
-
```

Comment:
avoid

Reply:
intentionally removed the line break from this method, let me know if you have any concerns